operatorconfig/driverconfig/powerstore/v2.5.0/node.yaml

```diff
@@ -94,26 +94,35 @@ spec:
                   apiVersion: v1
                   fieldPath: spec.nodeName
             - name: X_CSI_POWERSTORE_NODE_NAME_PREFIX
-              value: csi-node
+              value: <X_CSI_POWERSTORE_NODE_NAME_PREFIX>
             - name: X_CSI_POWERSTORE_NODE_ID_PATH
               value: /node-id
             - name: X_CSI_POWERSTORE_NODE_CHROOT_PATH
               value: /noderoot
             - name: X_CSI_POWERSTORE_TMP_DIR
-              value: var/lib/kubelet/plugins/csi-powerstore.dellemc.com/tmp
+              value: /var/lib/kubelet/plugins/csi-powerstore.dellemc.com/tmp
             - name: X_CSI_DRIVER_NAME
-              value: "csi-powerstore"
+              value: "csi-powerstore.dellemc.com"
             - name: X_CSI_FC_PORTS_FILTER_FILE_PATH
-              value: /etc/fc-ports-filter
+              value: <X_CSI_FC_PORTS_FILTER_FILE_PATH>
             - name: X_CSI_POWERSTORE_ENABLE_CHAP
-              value: "false"
+              value: "<X_CSI_POWERSTORE_ENABLE_CHAP>"
             - name: X_CSI_POWERSTORE_CONFIG_PATH
               value: /powerstore-config/config
             - name: X_CSI_POWERSTORE_CONFIG_PARAMS_PATH
               value: /powerstore-config-params/driver-config-params.yaml
             - name: GOPOWERSTORE_DEBUG
               value: "true"
             - name: X_CSI_HEALTH_MONITOR_ENABLED
+              value: "<X_CSI_HEALTH_MONITOR_ENABLED>"
+          volumeMounts:
```

Comment:
@kk3, please check if volumeMounts section is duplicated in this file

Reply:
taken care